### PR TITLE
Motor Details

### PR DESCRIPTION
**Pull request by AliShahSafdari**
- Add the Motorbike details

### CODE REVIEW

**Review by mohametalmeari — APPROVED** [review]
Hi @AliShahSafdari 
Great job creating the details page. This part of the project is complete and there is nothing else to say other than... it is time to merge. 
Congrats! 🏅🎉

File: src/components/Details.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          <p>
+            Model:
+            {motorbike.created_at}
+          </p>
+        </div>
+      </div>
+      <div className="row">
+        <Link to="/">Go to Home</Link>
+      </div>
+    </div>
```
Comment (WilhelmK109) [inline src/components/Details.js]: - You did well by implementing the `Go to Home` link. One of the requirements is that "In the details page, the user can click the "Reserve" button. Please kindly fix this.

**Review by WilhelmK109 — CHANGES_REQUESTED** [review]
Hi @AliShahSafdari,
Great job implementing most of the requirements for the Details page task. There are some issues that need improvement for you to move to the next task, but you are almost there.

AlmostThereReachGIF

Please see the comments below:

**Review by SaneMyburg — APPROVED** [review]
## STATUS: APPROVED 🟢
Hi @AliShahSafdari ,

Your project is complete! There is nothing else to say other than... it's time to merge it :
Congratulations! 🎉

#### (Highlights) Good Points: 👍
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Awesome job! Creating details page✔🥳 🥳
- This milestone's requirements have been met with success. 🎉 🎉 🎉 ✔
#### Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**